lib/compat/wordpress-6.0/class-gutenberg-rest-edit-site-export-controller.php

```diff
@@ -76,7 +76,10 @@ public function export() {
 			return $filename;
 		}
 
-		$theme_name = wp_get_theme()->get( 'TextDomain' );
+		$stylesheet = get_stylesheet();
+		$stylesheet_directories = explode( '/', $stylesheet );
+		$theme_name = end( $stylesheet_directories );
```

Comment:
Nitpick: Can we align these?

```suggestion
		$stylesheet             = get_stylesheet();
		$stylesheet_directories = explode( '/', $stylesheet );
		$theme_name             = end( $stylesheet_directories );
```